src/main/java/jenkins/plugins/git/traits/UserIdentityTrait.java

```diff
@@ -50,6 +51,7 @@ public UserIdentityTrait(UserIdentity extension) {
      * Our {@link hudson.model.Descriptor}
      */
     @Extension
+    @Symbol("gitUserIdentity")
```

Comment:
Would prefer that traits end with the suffix 'Trait' so that they don't prevent using that symbol for an extension that has the same name (like this one does).
```suggestion
    @Symbol("gitUserIdentityTrait")
```